CHANGELOG.md

```diff
@@ -16,7 +16,22 @@ Breaking changes:
 
 Features:
 
-- [#1336](https://github.com/rails-api/active_model_serializers/pull/1336) Added support for Grape >= 0.13, < 1.0
+- [#1378](https://github.com/rails-api/active_model_serializers/pull/1378) Change association blocks
+  to be evaluated in *serializer* scope, rather than *association* scope. (@bf4)
+    * Syntax changes from e.g.
+      `has_many :titles do customers.pluck(:title) end` (in #1356) to
+      `has_many :titles do object.customers.pluck(:title) end`
+- [#1356](https://github.com/rails-api/active_model_serializers/pull/1356) Add inline syntax for
```

Comment:
Oh... They're just missing from changelog.  How would you prefer i include them?

B mobile phone

> On Dec 31, 2015, at 12:05 PM, Lucas Hosseini <notifications@github.com> wrote:
> 
> In CHANGELOG.md:
> 
> > @@ -16,7 +16,22 @@ Breaking changes:
> >  
> >  Features:
> >  
> > -- [#1336](https://github.com/rails-api/active_model_serializers/pull/1336) Added support for Grape >= 0.13, < 1.0
> > +- [#1378](https://github.com/rails-api/active_model_serializers/pull/1378) Change association blocks
> > +  to be evaluated in *serializer* scope, rather than *association* scope. (@bf4)
> > +    * Syntax changes from e.g.
> > +      `has_many :titles do customers.pluck(:title) end` (in #1356) to
> > +      `has_many :titles do object.customers.pluck(:title) end`
> > +- [#1356](https://github.com/rails-api/active_model_serializers/pull/1356) Add inline syntax for
> Because this PR is #1378 and not #1356
> 
> —
> Reply to this email directly or view it on GitHub.
>